examples/cis3-nft-sponsored-txs/src/lib.rs

```diff
@@ -295,45 +256,45 @@ struct SetImplementorsParams {
 /// Part of the parameter type for the contract function `permit`.
 /// Specifies the message that is signed.
 #[derive(SchemaType, Serialize)]
-struct PermitMessage {
+pub struct PermitMessage {
     /// The contract_address that the signature is intended for.
-    contract_address: ContractAddress,
+    pub contract_address: ContractAddress,
     /// A nonce to prevent replay attacks.
-    nonce:            u64,
+    pub nonce:            u64,
     /// A timestamp to make signatures expire.
-    timestamp:        Timestamp,
+    pub timestamp:        Timestamp,
     /// The entry_point that the signature is intended for.
-    entry_point:      OwnedEntrypointName,
+    pub entry_point:      OwnedEntrypointName,
     /// The serialized payload that should be forwarded to either the `transfer`
     /// or the `updateOperator` function.
     #[concordium(size_length = 2)]
-    payload:          Vec<u8>,
+    pub payload:          Vec<u8>,
 }
 
 /// The parameter type for the contract function `permit`.
 /// Takes a signature, the signer, and the message that was signed.
 #[derive(Serialize, SchemaType)]
 pub struct PermitParam {
-    /// Signature/s. The CIS3 standard supports multi-sig accounts but because
-    /// the `public_key_registry` in this contract maps one public key to one
-    /// account, only one signature has to be provided for this contract. This
-    /// signature has to be at the key 0 in both maps below.
-    #[concordium(size_length = 1)]
-    signature: BTreeMap<u8, BTreeMap<u8, SignatureEd25519>>,
+    /// Signature/s. The CIS3 standard supports multi-sig accounts. But for
+    /// simplicity, this contract only supports signatures that were generated
```

Comment:
Is it still true that we only support accounts with one public key?

Reply:
Not true anymore. I updated the comment.